# https://github.com/util-linux/util-linux/issues/1405
# for unknown reason this fix is not included in the v2.37 branch

--- libuuid/src/unparse.c.origin	2021-05-06 10:55:46.910423300 +0000
+++ libuuid/src/unparse.c	2021-11-27 11:20:36.312285649 +0000
@@ -33,6 +33,7 @@
  */
 
 #include <stdio.h>
+#include "c.h"
 
 #include "uuidP.h"
 
